apps/ffdweb-site/src/app/not-found.tsx

```diff
@@ -1,10 +1,12 @@
 import { ErrorMessage } from '@filecoin-foundation/ui/ErrorMessage'
+import { NotFoundAnalytics } from '@filecoin-foundation/ui/NotFoundAnalytics'
 
 import { BASE_DOMAIN } from '@/constants/siteMetadata'
 
 export default function NotFound() {
```

Comment:
Could `NotFound` and `NotFoundAnalytics` be abstracted into the shared UI library as well?